Purpose: Use multiarch library search path to find and link with libcrypt
Author:  Roger Leigh
Fixes:   #629677
Status:  Not sent upstream.


--- a/src/Makefile
+++ b/src/Makefile
@@ -78,10 +78,15 @@ else
   SULOGINLIBS	=
 endif
 
+LIBDIR=/usr/lib
 # Additional libs for GNU libc.
 ifneq ($(wildcard /usr/lib*/libcrypt.a),)
+  SULOGINLIBS   += -lcrypt
+else
+ifneq ($(wildcard $(LIBDIR)*/libcrypt.a),)
   SULOGINLIBS	+= -lcrypt
 endif
+endif
 
 all:		$(BIN) $(SBIN) $(USRBIN)
 
